spark/v3.1/spark/src/main/java/org/apache/iceberg/spark/actions/BaseDeleteOrphanFilesSparkAction.java

```diff
@@ -86,6 +87,8 @@
     }
   }, DataTypes.StringType);
 
+  private static final ExecutorService DEFAULT_DELETE_EXECUTOR_SERVICE = null;
```

Comment:
Not sure I see a huge point to this variable (feel like a comment could have sufficed), but I guess it's there in the other version, so no matter :)

I guess the explicit assumption is that Tasks execute without a service, executes the task on a single thread.